admin/scripts/modules/uvod.php

```diff
@@ -191,6 +204,7 @@
 // form s osobními údaji
 if($uPracovni) {
   $udaje = [
+    'login_uzivatele'       =>  'Přezdívka',
     'jmeno_uzivatele'       =>  'Jméno',
     'prijmeni_uzivatele'    =>  'Příjmení',
     'ulice_a_cp_uzivatele'  =>  'Ulice',
```

Comment:
Aktuální review je jen na kód, testovat budu po případných úpravách.

DbDuplicateEntryException by byl ok způsob jak indikovat výjimku. Není to specifická výjimka ale naopak velmi přesně sedí na situaci "v databázi je něco duplicitního - nelze, chyba". Pouze to tam nikdo neimplementoval, nyní už je to potřeba. I v třídě uživatel je pak vidět, že se to odchytáva a specificka výjimka pro třídu uživatel jsou ty upravené `DuplicitniEmailException` a `DuplicitniLoginException`.

Reply:
Změnění commitu je ok, k komentářům se dá v případě nutnosti dostat kliknutím na "show outdated".